### DISCUSSION TITLE
Game Resources/Assets

### DISCUSSION THREAD

**Discussion by CatCode79** [Davidster/ikari#64] [General]
I open this thread to save links to resources and assets usable in the engine/game.

Music similar to Doom Eternal:
https://davidkbd.itch.io/eternity-metal-scfi-music-pack

**Comment by Davidster** [comment]
nice! it makes me also think that some credits should be written (in the readme?) for the assets used in the screenshots / videos. even though it's not a game yet, just a test scene it could be nice to give credit

**Comment by CatCode79** [comment]
Yes, I think that in the readme it could be fine, in the case of this asset apparently you can't use the music for commercial projects, so we could use them in the project examples.
